test/arithmetic/floating_point/butterfly_test.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+
+import 'dart:math';
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:rohd_hcl/src/arithmetic/floating_point/fft/butterfly.dart';
+import 'package:rohd_hcl/src/arithmetic/signals/floating_point_logics/complex_floating_point_logic.dart';
+import 'package:test/test.dart';
+
+ComplexFloatingPoint newComplex(double real, double imaginary) {
+  final realFP = FloatingPoint64();
+  final imaginaryFP = FloatingPoint64();
+
+  final realFPValue = FloatingPoint64Value.populator().ofDouble(real);
+  final imaginaryFPValue = FloatingPoint64Value.populator().ofDouble(imaginary);
+
+  realFP.put(realFPValue);
+  imaginaryFP.put(imaginaryFPValue);
+
+  final complex = ComplexFloatingPoint(
+    exponentWidth: realFP.exponent.width,
+    mantissaWidth: realFP.mantissa.width,
+  );
+  complex.realPart <= realFP;
+  complex.imaginaryPart <= imaginaryFP;
+
+  return complex;
+}
+
+class Complex {
```

Comment:
this class is defined in multiple files?

Reply:
you can create a directory in either lib (not exported) or in the test directory and can reference other files that way